content/en/examples/storage/storageclass/storageclass-ceph-rbd.yaml

```
@@ -0,0 +1,17 @@
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: fast
+provisioner: kubernetes.io/rbd
+parameters:
+  monitors: 10.16.153.105:6789
```

Optionally, switch it to a documentation IP address
```suggestion
  monitors: 198.19.254.105:6789
```